scripts/download-data.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/env python
```

Comment:
Is this meant to be part of this PR?

Reply:
The IERS wasn't downloading without it so the tests were failing.  I changed the description of PR.